.gitattributes

```diff
@@ -3,3 +3,5 @@
 # Undo the GitHub's default
 # https://github.com/github/linguist/blob/96ad1185828f44bb9b774328a584551ee57ed264/lib/linguist/vendor.yml#L177
 packages/**/*.d.ts -linguist-vendored
+/docs/** -linguist-documentation
```

Comment:
but this is documentation?

Reply:
I have used the following flow of thoughts:

What's the use case for the language stats? Developers want to known if the languages used by a project match the ones he wants to use. Most of the documentation is markdown and UI demos developers will copy and paste in their projects.

Reply:
> Developers want to known if the languages used by a project match the ones they want to use.

What's the evidence that people look at the github repository for that? That seems like a pretty flawed thought process for a mono repository.

Reply:
> What's the evidence that people look at the github repository for that?

I have asked @mbrookes what he was using the stats for 😁. What are you using them for?

Reply:
Because you asked me what I've used it for, that now makes it "the primary use case for the language stat"? 🤣 

I also commented on how it's flawed. 🤦